client/sync.js

```diff
@@ -95,6 +95,19 @@ const startSync = (requester) => {
       })
     })
   })
+  ipc.on(messages.FETCH_SYNC_DEVICES, (e) => {
+    logSync(`fetching devices`)
+    requester.list(proto.categories['PREFERENCES'], 0).then((s3Objects) => {
+      const records = requester.s3ObjectsToRecords(s3Objects)
+      logSync(`fetched ${records.length} devices`)
+      const jsRecords = records.map((record) => {
+        return recordUtil.syncRecordAsJS(record)
+      })
+      const lastObject = s3Objects[s3Objects.length - 1]
+      const lastRecordTimestamp = s3Helper.parseS3Key(lastObject.Key).timestamp
+      ipc.send(messages.GET_EXISTING_OBJECTS, 'PREFERENCES', jsRecords, lastRecordTimestamp)
```

Comment:
it makes slightly more sense to me if this were `RESOLVED_SYNC_RECORDS` instead of `GET_EXISTING_OBJECTS`, since sync doesn't appear to need or use the existing objects in this case.